swim/test_utils.go

```diff
@@ -134,7 +134,7 @@ func memberlistHasMembers(t *testing.T, m *memberlist, members []Member) {
 	}
 }
 
-func bootstrapNodes(t *testing.T, testNodes ...*testNode) {
+func bootstrapNodes(t *testing.T, waitForConvergence bool, testNodes ...*testNode) []string {
```

Comment:
I'm not sure the `waitForConvergence` needs to be part of the bootstrap function, does it?

I think we could create a separate function, e.g. `waitForConvergence(timeout time.Duration, nodes ...*testNode)` and then call that if you have a test that needs this.

I think that has the advantage that you don't need change the existing `bootstrapNodes` calls and those calls are slightly more readable, i.e. you don't have a boolean in the call which requires you to look at the signature to figure out what it is.